pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -189,6 +189,10 @@ func (r *ReconcileSyncSetInstance) Reconcile(request reconcile.Request) (reconci
 		return r.syncDeletedSyncSetInstance(ssi, ssiLog)
 	}
 
+	// Assume applied is false until we successfully apply everything to cluster.
+	ssi.Status.Applied = false
```

Comment:
Since we are not doing the defer any more, this does not need to be done up here any more. We can keep the deep copy where it was before and omit the code that explicitly sets `Applied` to false.